sofiadinizms/vida-de-inseto
# Feature/balloons

## What I did
1. Added the correct Thai assets
2. Added the correct background and log assets
3. Added alert balloons to all the levels
4. Added correct alert balloon assets
5. Added the alert texts from the UX Writing item on the Kanban
6. Added the correct fruit assets

## How to test
1. Clone this branch
2. Run each level individually and check if all of the assets are correct
3. Check if every level has an alert balloon and if the alert has the proper text

## Technical debt
1. Fixing the level 6 circles
2. Figuring out what to do with the level 4 text about how much longer you need to wait
